test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1422,3 +1422,18 @@ firrtl.circuit "MemoryPrefixCopying" {
     } : !firrtl.bundle<addr: uint<6>, en: uint<1>, clk: clock, data flip: vector<uint<8>, 1>>
   }
 }
+
+
+firrtl.circuit "DiscardableAttributes" {
+  // COMMON-LABEL: firrtl.module @DiscardableAttributes
+  firrtl.module @DiscardableAttributes(in %clock: !firrtl.clock, in %reset: !firrtl.uint<1>, in %a: !firrtl.bundle<a: uint<1>>) {
+    // CHECK-NEXT: %node_a = firrtl.node %a_a {foo} : !firrtl.uint<1>
+    // CHECK-NEXT: %wire_a = firrtl.wire {foo = "bar"} : !firrtl.uint<1>
+    // CHECK-NEXT: %reg_a = firrtl.reg %clock {foo} : !firrtl.clock, !firrtl.uint<1>
+    // CHECK-NEXT: %regreset_a = firrtl.regreset %clock, %reset, %a_a {foo} : !firrtl.clock, !firrtl.uint<1>, !firrtl.uint<1>, !firrtl.uint<1>
```

Comment:
Mega-nit: consider removing the types to make this more readable in the `CHECK` lines.